cmd/argocd/commands/app_test.go

```diff
@@ -338,3 +341,83 @@ func TestFormatConditionSummary(t *testing.T) {
 		}
 	})
 }
+
+func TestPrintOperationResult(t *testing.T) {
+	t.Run("Operation state is empty", func(t *testing.T) {
+		output, _ := captureOutput(func() error {
+			printOperationResult(nil)
+			return nil
+		})
+
+		if output != "" {
+			t.Fatalf("Incorrect print operation output \"%s\", should be \"\"", output)
```

Comment:
Consider using %q instead of manually adding double quotes

Reply:
didnt know about it, thank you! fixed